src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.Forwards.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+// These types were moved to System.Primitives
```

Comment:
There is no assembly called `System.Primitives`

Reply:
I do not sure whether these "These types were moved to..." comments add any value. If you need to know where the type lives now, it is easy to find out.

Reply:
Sure, I can remove them. Also, I copied the comment from the existing ref.